runtime/src/iree/base/config.h

```diff
@@ -91,8 +91,9 @@ typedef IREE_HOST_SIZE_T iree_host_size_t;
 typedef IREE_DEVICE_SIZE_T iree_device_size_t;
 
 // Maximum representable value in iree_device_size_t.
-#define IREE_DEVICE_SIZE_MAX \
-  (sizeof(iree_device_size_t) == 4 ? UINT32_MAX : UINT64_MAX)
+#define IREE_DEVICE_SIZE_MAX                                         \
+  ((iree_device_size_t)(sizeof(iree_device_size_t) == 4 ? UINT32_MAX \
+                                                        : UINT64_MAX))
```

Comment:
Actually, this change is not needed for fixing the build. I made the change for consistency.